heroku/cli
# refactor(certs-v5): Move command `certs:update` to CLI

## Description

Moves the `certs:update` command to CLI, converting to OClif Core and TS.

We're removing here a `bypass` flag that wasn't in use. It didn't effect the command behavior in any way and the described function isn't necessary any longer.

We're also removing here all shared libraries and tests for the `certs-v5` commands, some of which were migrated to the feature branch. Two v5 commands are left untouched: `certs:chain` and `certs:key`, but they have no use and didn't used any of the libraries' functions.

## Test instructions

You can test by pulling this branch, installing dependencies with `yarn` and building with `yarn build`. Then run the following commands, knowing that if you specify both the `--name` and `--endpoint` flags, you should see an error.

```
./bin/run certs:update --help                                                  # always test that help looks ok.
./bin/run certs:update CRT KEY -a test_app_without_certs                       # shows error.
./bin/run certs:update CRT KEY -a test_app_single_cert                         # updates the certificate after confirmation.
./bin/run certs:update CRT KEY -a test_app_multiple_certs                      # shows error.
./bin/run certs:update CRT KEY -a test_app_multiple_certs --name CERT_NAME     # updates the certificate after confirmation.
```

You can generate a self signed certificate for testing with the following command:

```
heroku certs:generate DOMAIN -a test_app --selfsigned
```

## SOC2 Compliance
[GUS Work Item](https://gus.lightning.force.com/a07EE00001avkj9YAA)
